DatadogSessionReplay/Sources/SessionReplayConfiguration.swift

```diff
@@ -43,6 +49,23 @@ extension SessionReplay {
 
         /// Creates Session Replay configuration
         /// - Parameters:
+        ///   - sampleRate: The sampling rate for Session Replay. It is applied in addition to the RUM session sample rate.
+        ///   - touchPrivacyLevel: The way user touches (e.g. tap) should be masked. Default: `.hide`.
+        ///   - customEndpoint: Custom server url for sending replay data. Default: `nil`.
+        public init(
+            sampleRate: Float,
```

Comment:
Note: The parameter name was changed to `sampleRate` in this initializer to distinguish it from the other initializer. Without this change, this would result in an ambiguous call, and Swift not being able to determine which initializer to use.

Reply:
/change-request Lets not expose the initialiser until the feature is implemented.